packages/devtools_app/lib/src/shared/tree.dart

```diff
@@ -146,10 +144,10 @@ class TreeViewItem<T extends TreeNode<T>> extends StatefulWidget {
 
   final T data;
 
-  final Widget Function(VoidCallback onPressed) buildDisplay;
+  final Widget Function(VoidCallback onPressed)? buildDisplay;
 
-  final void Function(T) onItemSelected;
-  final void Function(T) onItemExpanded;
+  final void Function(T)? onItemSelected;
+  final void Function(T)? onItemExpanded;
```

Comment:
make all of these non-nullable and required.  Looks like the TreeViewItem class can also be private. Let's do that while we are here.

Reply:
done

Reply:
buildDisplay can be non-nullable too.

Reply:
done